ios/src/ImageCropPicker.m

```diff
@@ -505,32 +505,76 @@ - (NSString *)determineMimeTypeFromImageData:(NSData *)data {
         case 0x4D:
             return @"image/tiff";
     }
-    return @"";
+    return @"image/jpeg";
+}
+
+- (NSString *)determineExtensionFromImageData:(NSData *)data {
+    uint8_t c;
+    [data getBytes:&c length:1];
+    
+    switch (c) {
+        case 0xFF:
+            return @".jpg";
+        case 0x89:
+            return @".png";
+        case 0x47:
+            return @".gif";
+        case 0x49:
+        case 0x4D:
+            return @".tiff";
+    }
+    return @".jpg";
```

Comment:
why not in a `default` case?

Reply:
You are right... the switch case has a return for every case and then a final return at the end... 
I'll skip it for now as I don't want to hassle with the lib today. will fix in next iterations.